src/capital-project/capital-project.repository.ts

```diff
@@ -103,6 +105,67 @@ export class CapitalProjectRepository {
     }
   }
 
+  async findGeoJsonByManagingCodeCapitalProjectId(
+    params: FindCapitalProjectGeoJsonByManagingCodeCapitalProjectIdPathParams,
+  ): Promise<FindGeoJsonByManagingCodeCapitalProjectIdRepo> {
+    const { managingCode, capitalProjectId } = params;
+    try {
+      return await this.db
+        .select({
+          id: capitalProject.id,
+          managingCode: capitalProject.managingCode,
+          description: capitalProject.description,
+          managingAgency: capitalProject.managingAgency,
+          minDate: capitalProject.minDate,
+          maxDate: capitalProject.maxDate,
+          category: capitalProject.category,
+          sponsoringAgencies: sql<
+            Array<string>
+          >`ARRAY_AGG(DISTINCT ${agencyBudget.sponsor})`,
+          budgetTypes: sql<
+            Array<string>
+          >`ARRAY_AGG(DISTINCT ${agencyBudget.type})`,
+          commitmentsTotal: sum(capitalCommitmentFund.value).mapWith(Number),
+          geometry: sql<string | null>`
+            CASE
+            WHEN 
+              ${capitalProject.liFtMPoly} IS NOT null
+            THEN
+              ST_asGeoJSON(ST_Transform(${capitalProject.liFtMPoly}, 4326))	
+            ELSE
+              ST_asGeoJSON(ST_Transform(${capitalProject.liFtMPnt}, 4326))
+            END
+          `.as("geometry"),
+        })
+        .from(capitalProject)
+        .leftJoin(
+          capitalCommitment,
+          and(
+            eq(capitalProject.managingCode, capitalCommitment.managingCode),
+            eq(capitalProject.id, capitalCommitment.capitalProjectId),
+          ),
+        )
+        .leftJoin(
+          agencyBudget,
+          eq(agencyBudget.code, capitalCommitment.budgetLineCode),
+        )
+        .leftJoin(
+          capitalCommitmentFund,
+          eq(capitalCommitmentFund.capitalCommitmentId, capitalCommitment.id),
+        )
+        .where(
+          and(
+            eq(capitalProject.managingCode, managingCode),
+            eq(capitalProject.id, capitalProjectId),
```

Comment:
Because this query is based on the version 'capital project' endpoint with the double counting bug (#357), it also has the double-counting bug. It can be resolved with the same fix, which is to add a "capital commitment fund category filter":


```suggestion
            eq(capitalProject.id, capitalProjectId),
            eq(capitalCommitmentFund.category, "total"),
```

[Original bug fix](https://github.com/NYCPlanning/ae-zoning-api/pull/358/files)